src/main/java/wolox/training/dtos/BookDto.java

```diff
@@ -0,0 +1,82 @@
+package wolox.training.dtos;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BookDto {
+
+    private String title;
+    private String subtitle;
+    private List<AuthorDto> authors;
+    private int number_of_pages;
```

Comment:
We usually name object attributes with camelCase rather than snake_case 🙂

Reply:
Done :D x2 (?